contribs/drt/src/main/java/org/matsim/contrib/drt/passenger/DrtRequest.java

```diff
@@ -109,8 +111,8 @@ public Link getToLink() {
 	}
 
 	@Override
-	public Id<Person> getPassengerId() {
-		return passengerId;
+	public List<Id<Person>> getPassengerIds() {
+		return Collections.unmodifiableList(passengerIds);
```

Comment:
In this and other similar places, consider using the "more modern" Collections API, e.g. `List.copyOf()` (the created list is also unmodifiable).